src/GridFS/StreamWrapper.php

```diff
@@ -90,6 +93,30 @@ public static function register(string $protocol = 'gridfs'): void
         stream_wrapper_register($protocol, static::class, STREAM_IS_URL);
     }
 
+    /**
+     * Rename all revisions of a filename.
+     *
+     * @return bool True on success or false on failure.
+     */
+    public function rename(string $path_from, string $path_to): bool
```

Comment:
I realize these parameter names were taken from https://www.php.net/manual/en/streamwrapper.rename.php but do we want to use something more internally consistent with the library? This would match the format used in RenameCollection:

### 
```suggestion
    public function rename(string $fromPath, string $toPath): bool
```

Reply:
So simple, I hadn't thought of it. Fixed, and the phpcs rule exclusion can be removed.